src/main/java/uk/gov/hmcts/reform/civil/service/user/UserInformationService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.civil.service.user;
+
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.exceptions.CaseNotFoundException;
+import uk.gov.hmcts.reform.civil.service.CoreCaseUserService;
+import uk.gov.hmcts.reform.civil.service.UserService;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserInformationService {
+
+    Logger log = LoggerFactory.getLogger(UserInformationService.class);
```

Comment:
using the standard config for logging, it would be easier to annotate the class with Slf4j